app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -264,7 +265,8 @@ class ChatActivity : AppCompatActivity(), IChatView {
 
     //Replies user with Speech
     override fun voiceReply(reply: String, language: String) {
-        val audioFocus = getSystemService(Context.AUDIO_SERVICE) as AudioManager
+        val audioFocus = getSystemService(Context.AUDIO_SERVICE)
+        if (audioFocus !is AudioManager) return
```

Comment:
For getSystemService, it is fine to cast as exception on cast failure is accepted. After this change, error will silently fail